lib/ffmpeg/qsv/util.py

```diff
@@ -10,6 +10,9 @@
 def using_compatible_driver():
   return get_media()._get_driver_name() in ["iHD", "d3d11", "dxva2"]
 
+def have_encode_main10sp(encoder):
+  return try_call(f"{exe2os('ffmpeg')} -hide_banner -h encoder={encoder} | grep '\-main10sp'")
+
```

Comment:
Only ffmpeg qsv since it is the only plugin that supports this option.  Not sure if ffmpeg vaapi will support it later and/or use different option name/syntax.  We can move later if added to other plugins.

Reply:
main10sp is only a vpl feature.  Thus this isolation into qsv plugin.